bin/lhapdf.in

```diff
@@ -306,7 +306,7 @@ def download_url(source, dest_dir, dryrun=False):
                     dest_file.write(buffer)
 
                     status  = chr(13) + '%s: ' % os.path.basename(url)
-                    status += r'%s [%3.1f%%]' % (convertBytes(file_size_dl).rjust(10), file_size_dl * 100. / file_size)
+                    #status += r'%s [%3.1f%%]' % (convertBytes(file_size_dl).rjust(10), file_size_dl * 100. / file_size)
                     sys.stdout.write(status+' ')
```

Comment:
Does this still have to be commented out? See above.

Reply:
Good point. I've uncommented the lines that use "file_size" variable.